integration-tests/smoke/automation_test.go

```diff
@@ -101,101 +101,115 @@ func TestAutomationBasic(t *testing.T) {
 func SetupAutomationBasic(t *testing.T, nodeUpgrade bool) {
 	t.Parallel()
 	l := utils.GetTestLogger(t)
-	var (
-		upgradeImage   string
-		upgradeVersion string
-		err            error
-		testName       = "basic-upkeep"
-	)
-	if nodeUpgrade {
-		upgradeImage, err = utils.GetEnv("UPGRADE_IMAGE")
-		require.NoError(t, err, "Error getting upgrade image")
-		upgradeVersion, err = utils.GetEnv("UPGRADE_VERSION")
-		require.NoError(t, err, "Error getting upgrade version")
-		testName = "node-upgrade"
-	}
-	chainClient, chainlinkNodes, contractDeployer, linkToken, registry, registrar, onlyStartRunner, testEnv := setupAutomationTest(
-		t, testName, ethereum.RegistryVersion_2_0, defaultOCRRegistryConfig, nodeUpgrade,
-	)
-	if onlyStartRunner {
-		return
-	}
 
-	consumers, upkeepIDs := actions.DeployConsumers(
-		t,
-		registry,
-		registrar,
-		linkToken,
-		contractDeployer,
-		chainClient,
-		defaultAmountOfUpkeeps,
-		big.NewInt(automationDefaultLinkFunds),
-		automationDefaultUpkeepGasLimit,
-	)
-
-	l.Info().Msg("Waiting for all upkeeps to be performed")
-	gom := gomega.NewGomegaWithT(t)
-	gom.Eventually(func(g gomega.Gomega) {
-		// Check if the upkeeps are performing multiple times by analyzing their counters and checking they are greater than 10
-		for i := 0; i < len(upkeepIDs); i++ {
-			counter, err := consumers[i].Counter(context.Background())
-			require.NoError(t, err, "Failed to retrieve consumer counter for upkeep at index %d", i)
-			expect := 5
-			l.Info().Int64("Upkeeps Performed", counter.Int64()).Int("Upkeep ID", i).Msg("Number of upkeeps performed")
-			g.Expect(counter.Int64()).Should(gomega.BeNumerically(">=", int64(expect)),
-				"Expected consumer counter to be greater than %d, but got %d", expect, counter.Int64())
-		}
-	}, "5m", "1s").Should(gomega.Succeed()) // ~1m for cluster setup, ~2m for performing each upkeep 5 times, ~2m buffer
+	registryVersions := map[string]ethereum.KeeperRegistryVersion{
+		//"registry_2_0": ethereum.RegistryVersion_2_0,
```

Comment:
:scissors:

Reply:
Sync-ed with Anirudh on this, will keep 2.0 uncommented, and comment out v2.1 since 2.1 happy path is not fully working. With this, we can keep this PR small and land, will have new PRs for other features.